javascript/stack-and-queue/New Implementation/queue-imp.test.js

```diff
@@ -49,6 +49,6 @@ const Queue = require ("./queue-imp");
         let newQueue = new Queue();
         expect(()=>{
             newQueue.dequeue();
-        }).toThrow("queue is empty");
+        }).toThrow("this is empty");
```

Comment:
Replace with `"empty Queue"` because in your queue-imp.js, when it is null, it returns the string "empty Queue"